app/controllers/task_managements_controller.rb

```diff
@@ -20,7 +20,14 @@ def create
       session.delete(:searcher)
       flash.clear
       flash[:notice] = "Your taskee has been notified"
-      notify("taskee", @task.taskee_id)
+      tasker = User.find(@task.tasker_id).firstname
+      Notification.create(
+        message: "#{@task.task.name} task from #{tasker}",
+        sender_id: @task.tasker_id,
+        receiver_id: @task.taskee_id
+      ).update_attribute(:notifiable, @task)
+      taskee_unnotified_count = Notification.unnotified_count(@task.taskee_id)
```

Comment:
Looking at 
` taskee_unnotified_count = Notification.unnotified_count(@task.taskee_id)`
and
` Notification.notify(@task.taskee_id, taskee_unnotified_count, "new_task")`
they essentially need one attribute, taskee_id. Why not refactor these functions to slim the controller.